openapi-maven-plugin/src/main/java/io/github/kbuntrock/configuration/ApiConfiguration.java

```diff
@@ -16,6 +16,9 @@ public class ApiConfiguration extends CommonApiConfiguration {
 	@Parameter
 	private String filename = "spec-open-api.yml";
```

Comment:
The default filename extension still is ".yml" if the json format is chosen. This is misleading. In this case, the default filename should be "spec-open-api.json".
This require probably a bit of logic. I don't mind if you put this logic in the "mergeWithCommonApiConfiguration" function.